### ISSUE TITLE
pgcli - Beta Testing - auto-completion

### ISSUE THREAD

**Issue by amjith** [dbcli/pgcli#427]
I'm reaching out to you with a request to help us beta test a new completion algorithm. Previously, keywords, table names, column names etc were ordered alphabetically. Now they are ordered by the frequency of usage. The frequency is calculated based on the history. We hope this will improve the experience and reduce typing. We'd love to get some user feedback to polish this feature before release. 

This is already available in master branch. You can install pgcli from `master` as follows: 

```
pip install -U https://github.com/dbcli/pgcli/archive/master.zip
```

With the improved suggestion, you'll notice the keywords in the suggestion menu are no longer alphabetical. For example if you type `F`, you'll see the keyword `FROM` listed well above `FREEZE` or other seldom used keywords that start with `F`. This is the good part, but I'm sure we've overlooked other usage scenarios where it might not be suitable. 

I'm reaching out to you since you've all used pgcli and cared enough to provide feedback or send in PRs. I would love to get some feedback from dedicated users.

Best,
Amjith

@d33tah 
@johshoff 
@brettatoms 
@fpietka 
@droustchev 
@xalley 
@GMLudo 
@qwesda 
@CyberDem0n
@wulczer
@dbcli/pgcli-core

**Comment by johshoff** [comment]
That's awesome, @amjith! I'm currently off the postgres bandwagon for a while, so probably no testing from me in the near future.

**Comment by fpietka** [comment]
Busy day. I finally got off mysql for a while :grin:, so I'll test it starting tomorrow.

**Comment by amjith** [comment]
This is now available in v1.0. Please upgrade. 

```
pip install -U pgcli
```
